content/blog/posts/this-is-how-we-watchdog.md

```diff
@@ -0,0 +1,36 @@
+---
+title: This is how we watchdog
+date: 2020-12-21
+author: Eva Ruth Moravec
+---
+We frequently mention that we “watchdog” the data that we work with as much as possible, but I thought I’d peel that back a bit. What do TJI’s oversight efforts look like? 
+
+Each month, I get new data from the Texas Office of the Attorney General reflecting the previous month's reported deaths in custody and shootings of and by law enforcement officer. I add this new data to our master data sets. Overnight, bots look for changes to the data sets, which are then pushed through our processing pipeline. 
```

Comment:
GitHub is [moving away `master`](https://github.com/github/renaming/) to more inclusive language. Not sure if it's worth considering a different term for our data sets.

Reply:
thanks for this note!! I will rephrase.